zeno/server.py

```diff
@@ -172,6 +175,18 @@ def embed_exists(model: str):
     def project_embed_into_2d(req: EmbedProject2DRequest):
         return project_into_2d(zeno.df, zeno.id_column, req.model, req.column)
 
+    @api_app.post(
+        "/slice-finder-project", tags=["zeno"], response_model=SliceFinderMetricReturn
```

Comment:
This return type should be a class in classes/slice_finder.py or something like that.